File: C Language/Q30.c
#include <stdio.h>

int main() {
    int number;

    printf("Enter an integer: ");
    scanf("%d", &number);

    // Check remainder when divided by 2
    if (number % 2 == 0) {
        printf("%d is Even.\n", number);
    } else {
        printf("%d is Odd.\n", number);
    }

    return 0;
}

File: Python Language/Q30.py
# Program to check if a number is Odd or Even

number = int(input("Enter an integer: "))

# The modulo operator (%) returns the remainder.
# If remainder is 0 when divided by 2, it is Even.
if number % 2 == 0:
    print(f"{number} is Even.")

if number % 2 != 0:
    print(f"{number} is Odd.")